mne/io/cnt/tests/test_cnt.py

```diff
@@ -26,7 +26,8 @@ def test_data():
     with warnings.catch_warnings(record=True) as w:
         raw = _test_raw_reader(read_raw_cnt, montage=None, input_fname=fname,
                                eog='auto', misc=['NA1', 'LEFT_EAR'])
-    assert_true(all('meas date' in str(ww.message) for ww in w))
+    assert_true(all('meas date' in str(ww.message) or
+                    'number of bytes' in str(ww.message) for ww in w))
```

Comment:
does the auto mode work with our test file? in any case we should test setting n_bytes here and even test with the wrong number to see how it breaks.